scripts/test-backwards.sh

```diff
@@ -15,9 +15,9 @@ setup(){
         output=$2
 	    head -c $size </dev/urandom > $output
     }
-    random_bytes_file "2K"   "$TEST_FILES_DIR/small-upload-testfile"          # create 2kb file of random bytes (inline)
-    random_bytes_file "5M"   "$TEST_FILES_DIR/big-upload-testfile"            # create 5mb file of random bytes (remote)
-    random_bytes_file "128M" "$TEST_FILES_DIR/multisegment-upload-testfile"   # create 128mb file of random bytes (remote)
+    random_bytes_file "2K"   "$TEST_FILES_DIR/small-upload-testfile"        # create 2kb file of random bytes (inline)
+    random_bytes_file "5M"   "$TEST_FILES_DIR/big-upload-testfile"          # create 5mb file of random bytes (remote)
+    random_bytes_file "13M" "$TEST_FILES_DIR/multisegment-upload-testfile"  # create 13mb file of random bytes (remote)
```

Comment:
This file is special. It needs to be exactly 2 segments. Without the reduced segments size a 129M will work but the 128M file triggered an error doing the last release test. 129M is 3 segments. 128M is 2 segments + 1 inline segment. We need to cover that inline segment.